configs/stage/roles/ocm.json

```diff
@@ -12,6 +12,30 @@
         "tenant": "ocm"
       }
     },
+    {
+      "name": "OCM Idp Editor",
+      "display_name": "OCM Idp Editor",
+      "description": "Grants permission to edit idps",
+      "system": true,
+      "platform_default": false,
+      "version": 4,
```

Comment:
This will work but is there reason to start version with 4 ? (version 1 is usually used for new roles)